ops/model.py

```diff
@@ -1722,7 +1712,7 @@ def __getitem__(self, key: str) -> str:
         self._validate_read()
         return super().__getitem__(key)
 
-    def update(self, other: _SupportsKeysAndGetItem[str, str], **kwargs: str):
+    def update(self, other: typing.Any = (), /, **kwargs: str):
```

Comment:
**TIL** that typeshed doesn't even define `dict.update`

https://github.com/python/typeshed/blob/main/stdlib/builtins.pyi#L1029

So, all good here!